packages/unit-testing/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@oracle/netsuite-suitecloud-unit-testing",
+  "version": "1.0.0",
+  "author": "Quarks",
```

Comment:
This is an internal team name. Should be Oracle?

Reply:
I'll change it and add ```"author": "Oracle"``` in all the packages

Reply:
Or just remove it. I think I removed author from package.json in master branch because npm was not happy.